config/names/clown.txt

```diff
@@ -26,12 +26,11 @@ Pocket
 Razzle Dazzle
 Redshirt McBeat
 Ronnie Pace
-Scootaloo
```

Comment:
It's an MLP pony but I can't find any data indicating the name is actually copyright or trademarked.